### PR TITLE
Added test case to validate error event

### PR DESCRIPTION
**Pull request by Divya063**
- Validates error events are skipped when the error event is not configured for a resource in test config.
- Validate that error events are skipped when an event occurs for a namespace which is not added in resource_config
Fixes part of #340

### CODE REVIEW

File: test/e2e/notifier/error/error.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+			resource := utils.GVRToString(test.GVR)
+			// checking if error operation is skipped
+			isAllowed := utils.AllowedEventKindsMap[utils.EventKind{
+				Resource:  resource,
+				Namespace: "all",
+				EventType: config.ErrorEvent}] ||
+				utils.AllowedEventKindsMap[utils.EventKind{
+					Resource:  resource,
+					Namespace: test.Namespace,
+					EventType: config.ErrorEvent}]
```
Comment (PrasadG193) [inline test/e2e/notifier/error/error.go:56]: @Divya063 Let's use `utils.CheckOperationAllowed()` method we recently added to check if event allowed. Please rebase with the latest develop

Comment (Divya063) [reply test/e2e/notifier/error/error.go:56]: @PrasadG193 Sure!

Comment (Divya063) [reply test/e2e/notifier/error/error.go:56]: @PrasadG193 Done!

Comment (PrasadG193) [reply test/e2e/notifier/error/error.go:56]: @Divya063 did you miss committing the changes? I don't see this resolved

**Review by PrasadG193 — APPROVED** [review]

**Comment by chetanpdeshmukh** [comment]
Hello @Divya063 ,
Thanks again for the PR!

Thank you for contributing to BotKube. Could you please fill out [this form](https://docs.google.com/forms/d/e/1FAIpQLSdULT22oZlpHPMXKVEtV4fZ6R8vdGDC2iFvMhoG6IhZkK1SFA/viewform?usp=sf_link), so we can send the well deserved awesome swags :slightly_smiling_face:

Team BotKube

**Pull request merged**